docs/sources/configuration/_index.md

```diff
@@ -1980,23 +2023,23 @@ The `limits_config` block configures global and per-tenant limits in Loki.
 # CLI flag: -validation.enforce-metric-name
 [enforce_metric_name: <boolean> | default = true]
 
-# Maximum number of active streams per user, per ingester. 0 to disable.
-# CLI flag: -ingester.max-streams-per-user
-[max_streams_per_user: <int> | default = 0]
-
 # Maximum line size on ingestion path. Example: 256kb.
-# There is no limit when unset.
+# There is no limit when unset or set to 0.
 # CLI flag: -distributor.max-line-size
-[max_line_size: <string> | default = none ]
+[max_line_size: <string> | default = 0 ]
 
 # Truncate log lines when they exceed max_line_size.
 # CLI flag: -distributor.max-line-size-truncate
-[max_line_size_truncate: <boolean> | default = false ]
+[max_line_size_truncate: <bool> | default = false ]
```

Comment:
based on the generic placeholders section at the top, boolean values should use the placeholder `<boolean>`

```suggestion
[max_line_size_truncate: <boolean> | default = false ]
```